CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ### Changed defaults / behaviours
 
+- Fix `GOCACHE` environment variable settings when building debian source
+  package on PPA build environment.
 - Make `PS1` environment variable changeable via `%environment` section on
   definition file that used to be only changeable via `APPTAINERENV_PS1`
   outside of container. This makes container's prompt customizable.
```

Comment:
Since we will want this on the release-1.1 branch, please move this to the 'Bug fixes' section.  While you're at it, you can move the message about `PS1` there too, since that's already been moved to the release-1.1 branch.

Reply:
Thank you I added those two to Bug fixes section.